akka-sample-kafka-to-sharding-scala/build.sbt

```diff
@@ -1,8 +1,8 @@
 val AkkaVersion = "2.6.8"
-val AlpakkaKafkaVersion = "2.0.3"
+val AlpakkaKafkaVersion = "2.0.4"
 val AkkaManagementVersion = "1.0.5"
 val AkkaHttpVersion = "10.1.11"
-val KafkaVersion = "2.4.0"
+val KafkaVersion = "2.4.1"
```

Comment:
should we depend on kafka at all? isn't that a transitive dep?

Reply:
This is actually for bringing in embedded-kafka.  It should probably be called `EmbeddedKafkaVersion`.